configurator/config.py

```diff
@@ -30,16 +30,9 @@ def from_text(cls, text, parser, encoding='ascii'):
         :ref:`parser <parsers>`. If ``text`` is provided as :class:`bytes`, then
         the ``encoding`` specified will be used to decode it.
         """
-        if parser == 'toml' or getattr(parser, '__module__', '').startswith('toml'):
-            if isinstance(text, bytes):
-                stream = BytesIO(text)
-            else:
-                stream = BytesIO(text.encode('utf-8'))
-        else:
-            if isinstance(text, bytes):
-                text = text.decode(encoding)
-            stream = StringIO(text)
-        return cls.from_stream(stream, parser)
+        if isinstance(text, bytes):
+            text = text.decode(encoding)
```

Comment:
When is this `bytes`?

Reply:
are you looking at an older commit? There should be no changes in config.py any longer